src/java_bytecode/java_string_library_preprocess.h

```diff
@@ -43,11 +45,10 @@ class java_string_library_preprocesst:public messaget
   void initialize_conversion_table();
   void initialize_refined_string_type();
 
-  exprt code_for_function(
-    const irep_idt &function_id,
-    const code_typet &type,
-    const source_locationt &loc,
-    symbol_table_baset &symbol_table);
+  void get_all_function_names(id_sett &methods) const;
+
+  exprt
+  code_for_function(const symbolt &symbol, symbol_table_baset &symbol_table);
```

Comment:
given the name of the function the return type should rather be `codet`

Reply:
You named this function! I'll let you fix it in a subsequent PR.